test/StreamTest.php

```
@@ -67,12 +71,29 @@ public function testCanInstantiateWithStreamResource(): void
         $this->assertInstanceOf(Stream::class, $stream);
     }
 
+    /**
+     * @todo For version 3, this test should verify that a GdImage raises an  exception.
```

with v3 we should verify if either `string` or `resource` is passed, **everything** else should throw an exception.